scripts/classify-failures

```diff
@@ -186,15 +178,53 @@ ISSUES = [
     },
     {
         "description": "[1296] https://github.com/rhinstaller/kickstart-tests/issues/1296",
-        "first_grep": "ERR anaconda:Exception ignored in atexit callback:",
+        "first_grep": "ERR anaconda:Exception ignored in atexit callback",
     },
     {
         "description": "[1303] https://github.com/rhinstaller/kickstart-tests/issues/1303",
         "first_grep": "WARNING gnome-kiosk:Lost or failed to acquire name org.gnome.Mutter.ServiceChannel",
     },
+    {
+        "description": "[886] https://github.com/rhinstaller/kickstart-tests/issues/886",
+        "first_grep": "/dev/vda4 shouldn't be mounted at",
+    },
+    {
+        "description": "[1311] https://github.com/rhinstaller/kickstart-tests/issues/1311",
+        "first_grep": "LVMError.*'/com/redhat/lvmdbus1/.*object: Timeout was reached",
+    },
+    {
+        "description": "[1312] https://github.com/rhinstaller/kickstart-tests/issues/1312",
+        "first_grep": "org.fedoraproject.Anaconda.Modules.Storage:gi.overrides.BlockDev.MpathError: Process reported exit code 1",
+    },
+    {
+        "description": "[1313] https://github.com/rhinstaller/kickstart-tests/issues/1313",
+        "first_grep": "ERR anaconda:    display.setup_display(anaconda, opts)",
+    },
+    {
+        "description": "[1314] https://github.com/rhinstaller/kickstart-tests/issues/1314",
+        "first_grep": "CRIT kernel:virtio_net virtio1 enp1s0: NETDEV WATCHDOG: CPU: 0: transmit queue 0 timed out",
+    },
+    {
+        "description": "[1318] https://github.com/rhinstaller/kickstart-tests/issues/1318",
+        "first_grep": "anaconda:anaconda: display: X or window manager startup failed: systemd exited with status 1",
+    },
+    {
+        "description": "[1319] https://github.com/rhinstaller/kickstart-tests/issues/1319",
+        "first_grep": "Anaconda.Modules.Payloads:ERROR:dnf.rpm:Error in POSTIN scriptlet in rpm package",
```

Comment:
I suggest to have the whole error message here to make sure it matches only the error caused by the `ca-certificates` package. To make sure a problem in a different package is not overlooked.

Reply:
It would mean to split out the problem for selinux-policy-targeted package from the https://github.com/rhinstaller/kickstart-tests/issues/1319 to a separate issue, right?

Reply:
Yes, I think they should should be separate issues - they would be most likely fixed in different components.

Reply:
So I rather removed [1319] from this PR.
I will watch the results some time and decide then if https://github.com/rhinstaller/kickstart-tests/issues/1319 should be really split.